libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceWebViewClientHelper.java

```diff
@@ -147,8 +146,8 @@ private static String isLoginRedirect(String url) {
     	final String ec = params.get("ec");
     	int ecInt = (ec != null ? Integer.parseInt(ec) : -1);
     	final String startURL = params.get("startURL");
-        if ((ecInt == HttpStatus.SC_MOVED_PERMANENTLY
-    			|| ecInt == HttpStatus.SC_MOVED_TEMPORARILY)
+        if ((ecInt == HttpURLConnection.HTTP_MOVED_PERM
```

Comment:
Can't use the org.apache.http.HttpStatus constants.
Fortunately, they are also defined in HttpURLConnection.